src/effects/effects_base/ieffectexecutionscenario.h

```diff
@@ -24,5 +24,7 @@ class IEffectExecutionScenario : MODULE_EXPORT_INTERFACE
     virtual muse::async::Notification lastProcessorIsNowAvailable() const = 0;
     virtual muse::async::Channel<EffectId> lastProcessorIdChanged() const = 0;
     virtual muse::Ret repeatLastProcessor() = 0;
+
+    virtual muse::Ret previewEffect(const EffectInstanceId& effectInstanceId, EffectSettings& settings) = 0;
```

Comment:
`settings` arg can't be const?

Reply:
I would like to, but I don't possible now 

```
muse::Ret EffectsProvider::previewEffect(AudacityProject&, Effect* effect, EffectSettings& settings)
{
    std::shared_ptr<SimpleEffectSettingsAccess> pAccess = std::make_shared<SimpleEffectSettingsAccess>(settings);
    EffectPreview(*effect, *pAccess.get(), nullptr, false);
```

SimpleEffectSettingsAccess - there is no constructor accepting `const EffectSettings&`